### PR TITLE
Cypress tests

### PR DESCRIPTION
**Pull request by arjandata**
Fixed all Cypress testst and updated to the Cypress 4.4

### CODE REVIEW

**Comment by jonkoops** [comment]
@arjandata could you squash your commits?

**Comment by royderks** [comment]
There's also a process in the build to run the files in `docker-compose.yml` and `Jenkinsfile`. Did you take into account of re-adding these commented out steps?

**Comment by jonkoops** [comment]
> There's also a process in the build to run the files in `docker-compose.yml` and `Jenkinsfile`. Did you take into account of re-adding these commented out steps? @arjandata

Yes, I would like to see this integrated into our CI as well :+1:

File: test/e2e/cypress-watch.js [hunk 1]
```diff
@@ -6,11 +6,4 @@ cypress.open({
   config: {
     integrationFolder: SCRIPTS_FOLDER,
   },
-  env: {
```
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply test/e2e/cypress-watch.js:9]: Should be removed 👍

File: test/e2e/cypress/integration/authentication_spec.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ describe('auth module', () => {
       cy.get(HEADER_MENU.rootMobile)
         .click()
         .get(`${HEADER_MENU.rootMobile} ${HEADER_MENU.login}`)
-        .contains('atlas.emp...')
+        .contains('n.druif@a...')
```
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply test/e2e/cypress/integration/authentication_spec.js:10]: pfff hahaha

File: test/e2e/package.json [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
   "scripts": {
-    "start": "cross-env API_ROOT=https://acc.api.data.amsterdam.nl node ./cypress-run.js",
-    "watch": "cross-env API_ROOT=https://acc.api.data.amsterdam.nl node ./cypress-watch.js"
+    "start": "node ./cypress-run.js",
+    "watch": "node ./cypress-watch.js"
   },
   "dependencies": {
-    "bluebird": "3.5.5",
-    "cross-env": "^5.2.0"
+    "bluebird": "3.7.2",
```
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply test/e2e/package.json:18]: Tried to remove this already, it's now really needed in cypress-run.js. Perhaps you have any suggestions on how we can replace this?

**Comment by chrisvanmook** [comment]
@arjandata I've squashed the commits, please `git pull --rebase`

File: src/app/pages.js [hunk 4]
```diff
@@ -41,19 +41,19 @@ const PAGES = {
```
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply src/app/pages.js]: Okay fixed it :P

File: test/e2e/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jonkoops) [reply test/e2e/package.json:18]: Yes, we should be able to use `util.promisify` to convert it to a Promise. Or are we using very specific Bluebird APIs?

See: https://nodejs.org/dist/latest-v12.x/docs/api/util.html#util_util_promisify_original

[the remark this answers is not in the document]
Comment (chrisvanmook) [reply test/e2e/package.json:19]: Nice one! Didn't knew about this. Updated

**Comment by jonkoops** [comment]
Merged in 1713d1cd26152088edcfec85bcd09da7b7d5a97f

**Pull request closed without merging**